src/client/client.go

```diff
@@ -47,6 +47,8 @@ type APIClient struct {
 var (
 	//BlockSize is used internally by PFS blockserver
 	BlockSize = 8 * 1024 * 1024 // 8 Megabytes
+	// MaxBlockSize specifies the maximum block size for any data type
+	MaxBlockSize = 100 * 1024 * 1024 // 100 MB
 	// MaxMsgSize is used to define the GRPC frame size, which we need to be greater than a block
 	MaxMsgSize = 3 * BlockSize
```

Comment:
Fair question.

Originally I set it higher than a block for two reasons:

1) the `MaxMsgSize` includes all of the payload of the grpc request including headers, etc (so if you set it to exactly the max size of a block and you tried to put/get a block of this size grpc would hangup)
2) we don't strictly enforce the block maximum. After the amount of data written to a block exceeds this size, we cut if off. Each 'write' is determined by the delimiter type. So for instance we could have a pathological case where the last line written itself was dozens of MBs. I created an issue for this case in #1280 

[However, the change that just landed (where we buffer our writes over grpc) should guarantee that we never put more than `MaxMsgSize/2` data in a putfile request](https://github.com/pachyderm/pachyderm/blob/master/src/client/pfs.go#L740). And skimming the server's `GetFile` code (and verifying this by testing), we write to the `StreamingBytesServer` in chunks of 4MB (`bufio`'s default buffer size).

So. If we want ... we could really make this any value over 4MB that leaves 'headroom' for the message headers/metadata.

I can make that change in a separate PR if you'd like. (Also I think that the pathological case above - #1280 - is invalid now that we buffer our writes).